sdks/python/apache_beam/transforms/combine_globally_test.py

```diff
@@ -0,0 +1,55 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
we could but `combiners_test.py` has ~1000 lines now. I would rather create small python files if possible.

Reply:
fair. although now combineglobally tests scenarios would be in multiple files. since combiners_test also include combine_globally tests. no strong opinion. on this, fine with me either way.